storage/buckets/sharing/index.md

```diff
@@ -22,7 +22,7 @@ any tables you create in the shared bucket become immediately available in the d
 You can specify who can link the bucket to a project:
 
 - **Organization Member** --- Only organization members are able to link the shared bucket to a project.
-- **Project Member** --- Every project member is able to link the shared bucket to a project.
+- **Project Member** --- Every project member in the organisation is able to link the shared bucket to a project.
```

Comment:
so is it the member of project or the member of organization? if the latter, how is it different from the first sharing option?

Reply:
exactly, then i don't get why i now reads:

> - **Project Member** --- Every project member in the organisation is able to link the shared bucket to a project.

Reply:
I just wanted to emphasize that project members from *other* organizations can not access it.

Reply:
ah!

Reply:
So maybe?

- Project Member --- Any project member under this organisation can link the shared bucket to their project

Reply:
hm, i've come up with this:

- **Project Member** --- To be able to link the bucket to a project, a person must be a user of the destination project (that project must be in the same organization as the source project).
- **Organization Member** --- To be able to link the bucket to a project, a person must be a user of the destination project 
(that project must be in the same organization as the source project) *and also a member of that organization*.

but not really happy with it, perhaps @hhanova will help? :)

Reply:
I think that's an improvement!  if not a little verbose :).  It's really amazing how difficult it is